OpenTabletDriver.Desktop/Diagnostics/LinuxEtcOSRelease.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace OpenTabletDriver.Desktop.Diagnostics
+{
+    public class LinuxEtcOSRelease
+    {
+        public const string OS_RELEASE_PATH = "/etc/os-release";
+
+        // keys that will never help with diagnostics
+        static readonly string[] IGNORED_KEYS = {
+            "ANSI_COLOR",
+            "LOGO"
+        };
+
+        public IDictionary<string,string> Raw;
```

Comment:
```suggestion
        public IDictionary<string, string> Raw;
```

Been a while since I touched C#, but I think this is the correct style?